### ISSUE TITLE
Add HighContrast theme to comply with accesibillity requirements!

### ISSUE THREAD

**Issue by hansgru** [highcharts/highcharts#2309]
Please add support for a HighContrast theme to comply with the accessibility requirements ( WCAG 2.0 Contrast Support ).

The existing Gray Theme is a very good starting point.

See how others did it:
http://docs.kendoui.com/tutorials/accessibility/high-contrast-theme
(contains the links to the official docs too)

or how it can look in action (select the HighContrast Theme from the builder):
http://demos.kendoui.com/themebuilder/web.html

regards.

**Comment by ipip2005** [comment]
Yeah, I would love this enhancement too. Also, there are some UI missing in high contrast mode. It might be large work but would improve a lot to a11y of Highcharts.

**Comment by TorsteinHonsi** [comment]
@oysteinmoseng FYI. A simple theme to go with the Accessibility features would be nice.

**Comment by oysteinmoseng** [comment]
We should also have a dark theme with good contrasts.

**Comment by oysteinmoseng** [comment]
Hold it there Ståle - let's keep this open as we want to introduce high contrast themes with fewer colors.
